src/components/LCACalculator/ElementImpactTable.tsx

```
@@ -669,21 +774,54 @@ const ElementImpactTable: React.FC<ElementImpactTableProps> = ({
+                          group={subGroup}
+                          isExpanded={expandedEbkp.includes(subGroup.code)}
+                          onToggle={() => toggleEbkpGroup(subGroup.code)}
+                          outputFormat={outputFormat}
+                          displayMode={displayMode}
+                          ebfNumeric={ebfNumeric}
+                        />
+                      ))}
+                  </React.Fragment>
+                ))}
+              </>
+            )}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)